.eslintrc

```diff
@@ -7,7 +7,18 @@
   "rules": {
     "func-names": 0,
     "id-length": [1, {"exceptions": ["$"]}],
-    "new-cap": [2, {"capIsNewExceptions": ["Deferred"]}]
+    "new-cap": [2, {"capIsNewExceptions": ["Deferred"]}],
+    "max-len": 0,
+    "no-prototype-builtins": 0,
+    "import/extensions": 0,
+    "import/prefer-default-export": 0,
+    "no-underscore-dangle": 0,
+    "no-useless-escape": 0,
+    "class-methods-use-this": 0,
+    "no-param-reassign": 0,
+    "no-restricted-syntax": 0,
+    "consistent-return": 0,
+    "array-callback-return": 0
```

Comment:
I added this rules, because`eslint-config-airbnb` changed a lot in the last 11 major releases 😉 Will fix the remaining rules in another PR